packages/fxa-content-server/tests/functional/lib/helpers.js

```diff
@@ -102,26 +101,11 @@ const takeScreenshot = function() {
         const rando = crypto.randomBytes(4).toString('hex');
         mkdirp.sync('/home/circleci/screenshots');
         fs.writeFileSync(`/home/circleci/screenshots/${rando}.png`, buffer);
-        console.log(`Screenshot saved to screenshots/${rando}.png`);
-      }
-      const screenCaptureHost = 'https://screencap.co.uk';
-      return got
-        .post(`${screenCaptureHost}/png`, {
-          body: buffer,
-          followRedirect: false,
-        })
-        .then(
-          res => {
-            console.log(
-              `Screenshot saved at: ${screenCaptureHost}${res.headers.location}`
-            );
-          },
-          //eslint-disable-next-line handle-callback-err
-          err => {
-            console.error('Capturing base64 screenshot:');
-            console.error(`data:image/png;base64,${buffer.toString('base64')}`);
-          }
+        // 36831081 is the repo number I think, but it's not available as an env var.
```

Comment:
Huh, no kidding! I could have sworn this number changed all the time. But, it seems to be stable between branches on the main repo and personal forks. Nice job sleuthing that out, last I looked I thought it would need a CircleCI API call to get this link!